include/kunit

```diff
@@ -186,9 +186,13 @@ need_kconfig:
 - SND_TEST_COMPONENT: m
 - SCF_TORTURE_TEST: m
 - REED_SOLOMON_TEST: m
+- NITRO_ENCLAVES: m
+- PCI: y
+- HAVE_PCI:y
```

Comment:
these need be moved to be together with NITRO_ENCLAVES_MISC_DEV_TEST.

Reply:
ok!moved "PCI" and  "HAVE_PCI" to be together with "NITRO_ENCLAVES_MISC_DEV_TES"